### DISCUSSION TITLE
I Need help

### DISCUSSION THREAD

**Discussion by electronzada7** [UnamSanctam/UnamWebPanel#384] [General]
![Screenshot_1](https://github.com/user-attachments/assets/39fa223d-36ea-44d8-8965-cb1064490082)
Can anyone solve my problem?

**Comment by UnamSanctam** [comment]
Can you post your pool configuration (pool, port and such? Since that message means that those miners can't contact the pool server.

**Reply by electronzada7** [reply]
"pool": "pool.supportxmr.com",
    "port": 443,

**Reply by electronzada7** [reply]
what other settings would you want to see?

**Reply by UnamSanctam** [reply]
Since you're using port 443 then don't forget to enable SSL as well, and SupportXMR also requires a worker name inside the password field. Also keep in mind that SupportXMR bans quite frequently.

**Reply by electronzada7** [reply]
What do you recommend I do to resolve it?

**Reply by UnamSanctam** [reply]
Enable SSL and add a worker name into the password field if you haven't already. If you're worried about getting banned on SupportXMR then you can use some other pool like nanopool or 2miners (though you'd have to mine Zephyr on 2miners instead since they removed Monero in favor of that).

**Reply by electronzada7** [reply]
and this pool here?
gulf.moneroocean.stream",
    "port": 20128,

**Reply by UnamSanctam** [reply]
You can use MoneroOcean, enable SSL and put a worker name in the password field on that pool too.

**Reply by electronzada7** [reply]
I think this is the one I'm using

**Reply by electronzada7** [reply]
![Screenshot_1](https://github.com/user-attachments/assets/383a3ae3-f894-47bc-96a6-c151e2024d8f)
wow thank you so much bro

**Reply by UnamSanctam** [reply]
Great, no problem.
